### PR TITLE
USHIFT-3781: Merge build_rpms.sh and create_local_repo.sh

### PR DESCRIPTION
**Pull request by vanhalenar**

### CODE REVIEW

File: test/bin/build_rpms.sh [hunk 1]
```diff
@@ -9,70 +9,112 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
...
-fi
-
-if ! ${BUILD_OK} ; then
-    echo "RPM build failed"
-    exit 1
-fi
-
-MAX_RUNTIME=$(jq -s 'max_by(.JobRuntime) | .JobRuntime | floor' "${BUILD_RPMS_LOG}")
-echo "RPM build complete in ${MAX_RUNTIME}s"
+build_rpms() {
```
Comment (ggiguash) [inline test/bin/build_rpms.sh:12]: When moving code from MAIN to a function, we need to take care of variables:
- Declare local variables explicitly using `local` or `local -r` for read-only
- UPPER case variables are assumed to be global

File: test/bin/build_rpms.sh [hunk 2]
```diff
@@ -9,70 +9,112 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
...
+        exit 1
+    fi
+
+    MAX_RUNTIME=$(jq -s 'max_by(.JobRuntime) | .JobRuntime | floor' "${BUILD_RPMS_LOG}")
+    echo "RPM build complete in ${MAX_RUNTIME}s"
+}
+
+make_repo() {
+    local repodir="$1"
+    local builddir="$2"
```
Comment (ggiguash) [inline test/bin/build_rpms.sh:84]:
```suggestion
    local -r repodir="$1"
    local -r builddir="$2"
```

File: test/bin/build_rpms.sh (hunk 1 above)
Comment (vanhalenar) [reply test/bin/build_rpms.sh:12]: My bad. Fixed

File: test/bin/build_rpms.sh (hunk 2 above)
Comment (vanhalenar) [reply test/bin/build_rpms.sh]: Done

**Pull request merged**